project/db/migrate/20221226034219_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string      :contributor, index: true
+      t.string      :comment, null: false
```

Comment:
@daiphone0228 
[must]

```suggestion
      t.text      :comment, null: false
```

Reply:
すみません、こちら漏れてましたね…対応しました。